UI/admin-request.html

```diff
@@ -0,0 +1,114 @@
+</head>
+
+<body>
+    <div class="backdrop"></div>
+    <div class="background-image"></div>
+    <header class="main-header">
+        <div>
+        <!-- Mobile Nav button -->
+        <button class="toggle-button">
+            <span class="toggle-button__bar"></span>
+            <span class="toggle-button__bar"></span>
+            <span class="toggle-button__bar"></span>
+        </button>
+            
+        </div>
+        <nav class="main-nav">
+            <ul class="main-nav__items">
+                <li class="main-nav__item">
+                    <a href="index.html">Home</a>
+                </li>
+                <li class="main-nav__item">
+                    <a href="login.html">Login</a>
+                </li>
+                <li class="main-nav__item">
+                    <a href="create-request.html">Services</a>
+                </li>
+                <li class="main-nav__item item-cta">
+                    <a href="siginup.html">Get Started</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <!-- Side nav for mobile -->
+  <nav class="mobile-nav">
+        <ul class="mobile-nav__items">
+            <li class="mobile-nav__item">
+                <a href="index.html">Home</a>
+            </li>
+            <li class="mobile-nav__item">
+                <a href="login.html">Login</a>
+            </li>
+            <li class="mobile-nav__item">
+                <a href="create-request.html">Services</a>
+            </li>
+            <li class="mobile-nav__item mobile-nav__item--cta">
+                <a href="siginup.html">Get Started</a>
+            </li>
+        </ul>
+      </nav>
+    <main>
+        <h1 class="request-list">REQUEST LIST</h1>
+        <section class="package" id="plus">
+            <a href="#">
+                <h1 class="package-title">David</h1>
+                <h2 class="badge">RESOLVED</h2>
+                <button type="submit" class="button">Approved</button>
+                <button type="submit" class="button">Disapproved</button>
+                <h2 class="sub-title">Work Description</h2>
+                <p class="page-info">is a fullstack javascript web application that provides users with the ability to reachout to the operations or
+                 repairs department regarding repair or maintenance requests and monitor the status of their request.</p>
+            </a>
+        </section>
+        
+        <section class="package" id="plus">
+            <a href="#">
+                <h1 class="package-title">Joseph</h1>
+                <h2 class="badge">RESOLVED</h2>
+                <button type="submit" class="button">Approved</button>
+                <button type="submit" class="button">Disapproved</button>
+                <h2 class="sub-title">Work Description</h2>
+                <p class="page-info">Get started immediately at zero cost!</p>
+            </a>
+        </section>
+        <div class="clearfix"></div>
```

Comment:
nitpick: On modern designs with the introduction of flexbox and grid we normally have no need for this. However, I would like to understand why you feel you needed it.